src/onyx/static/helpful_job_errors.clj

```diff
@@ -0,0 +1,858 @@
+(ns onyx.static.helpful-job-errors
+  (:require [clojure.string :refer [split join]]
+            [onyx.information-model :refer [model]]
+            [clj-fuzzy.metrics :refer [levenshtein]]
+            [io.aviso.ansi :as a]))
+
+(def manual-ex (ex-info "Manual validation check failed" {}))
+
+(def structure-names
```

Comment:
should we just rename these in the information model? The mapping is a bit awkward and we might as well just name them what they should be. That said, catalog-entry / task-map both make more sense than :catalog because the catalog is the full list of task-maps